var/spack/repos/builtin/packages/dbcsr/package.py

```diff
@@ -124,14 +130,22 @@ def cmake_args(self):
             "-DUSE_SMM=%s" % ("libxsmm" if "smm=libxsmm" in spec else "blas"),
             self.define_from_variant("USE_MPI", "mpi"),
             self.define_from_variant("USE_OPENMP", "openmp"),
-            # C API needs MPI
-            self.define_from_variant("WITH_C_API", "mpi"),
+            "-DWITH_C_API=false",
```

Comment:
I would remove it entirely because this option is treated at line 140.

Reply:
agreed, and this controls the C API entirely from the `mpi` variant, which is not what we want

Reply:
I would rather argue we should be adding a new variant `c-api` with the default set to enable, and then add a conflict for it when building with Fujitsu.

Reply:
@mtaillefumier  @dev-zro
My apologies for the late response. I've removed it.

Reply:
@dev-zero 
I've created a new variant `c-api`
It's enabled by default and conflicts when built with the Fujitsu compiler or when MPI is disabled."